test/app-luatest/console_debugger_session_test.lua

```diff
@@ -298,25 +298,38 @@ local function run_debug_session(cmdline, sequence, tmpfile, header)
                 debuglog('Execute command: "'..trim(cmd)..'"')
             end
 
-            local result
-            local clean_cmd = trim(cmd)
-            -- there should be empty stderr - check it before stdout
-            local errout = fh:read({ timeout = 0.05, stderr = true})
-            debuglog('stderr output: ', trim(errout))
-            if first and errout then
-                -- we do not expect anything on stderr
-                -- with exception of initial debugger header
-                t.assert_str_contains(trim(errout), header, false)
+            if first then
```

Comment:
I see no direct relation of the test changes to the code changes. Please, either extract it into its own commit with a proper description of the solved problem or make this relation clear in the commit message.

Reply:
Done.
>  The according test had to be ammended due to flaks on macOS.

Reply:
Does it mean that the test starts to sporadically fail on macOS after the given changes in the code? I can't understand it from the comment. If it is so, I'm OK to place it into the same commit. However, if we would stabilize the test case first, it would look better, IMHO.

`checkpatch` doesn't impose on the `NO_TEST` annotation due to these changes. However, we know that the new behavior is left untested. I don't insist on testing it (it is cumbersome), but, please, leave the annotation with a reason.

Nit: there is no noun 'flak', at least it is not the same as adjective 'flaky'.

Reply:
Removed the line and moved all test changes in separate commit.